client/scripts/controllers/app/login.ts

```diff
@@ -27,45 +27,44 @@ function createAccount(account: Account<any>) {
   });
 }
 
-export async function setActiveAccount(account: Account<any>) {
```

Comment:
I was doing a review with Jake of some of the async code, we thought it would be good to tidy up as we went. Just took a minute—fine with reverting if you prefer the old structure.

Reply:
Okay, seems fine to keep this to me. Just wondering why the change!